### ISSUE TITLE
[Firefox Nightly/Account Issue] Cannot send sub-50K views video, even though reputation is listed as 10

### ISSUE THREAD

**Issue by TteokbokkiNari** [videosegments/videosegments#21]
Tried in expert/normal, and still same issue. 
Also tried logging out and then back in, still facing the same issue. 

![firefox_2018-05-17_21-30-46](https://user-images.githubusercontent.com/6826910/40199848-ff5dd0dc-5a1a-11e8-9343-3417beefc924.png)

**Comment by videosegments** [comment]
I can't reproduce this bug. Can you please send video ID you trying to send? You can send me it on my email videosegmentsdev@gmail.com or write here.

**Comment by TteokbokkiNari** [comment]
It's the same for every video, but just tried this one: https://www.youtube.com/watch?v=YyWG1x3j5EY

I could give you my account log-in info if it might help diagnose the issue.

**Comment by videosegments** [comment]
I can't reproduce this error in normal/beta/nightly/developer versions of firefox. I've tried to send video with my account, second account with real score and third account with fake score. All three can send.
And it is odd that you see alert message instead of blue popup. Maybe you using outdated version of VideoSegments? Last version is 1.9.2.

**Comment by TteokbokkiNari** [comment]
Hm, I am using 1.9.2.
Will do some testing now, but I found out why the alert message is like that; will file another bug for that! 

This bug was a bitch to find, and I think it's more of a Firefox bug maybe.
I did everything I could until I decided to re-install addon as a last resort, and surprise, it works.. 
I feel stupid for spending an hour troubleshooting now, lol..

Sorry for causing trouble.
